docs: Update Docker instructions (#352)
ConduitIO/conduit: README.md

### Docker

Our Docker images are hosted on GitHub's Container Registry. To run the latest Conduit version, you should run the following command:

```
docker run -p 8080:8080 ghcr.io/conduitio/conduit:latest
```

> The Docker image includes the [UI](#ui), it is accessible if you navigate to `http://localhost:8080/ui`.

Reviewer: Just a suggestion.
```suggestion
The Docker image includes the [UI](#ui), you can access it by navigating to `http://localhost:8080/ui`.
```
